image-create

```diff
@@ -126,6 +128,9 @@ class MachineBuilder:
     def build(self):
         self.bootstrap_system()
 
+        if "boot" == self.machine.image:
+            return
```

Comment:
This will never happen. It's also not necessary, as the code checks for the existence of a `.setup` script next, and as there (rightfully) isn't one, it returns.

Reply:
Removed.